src/coreclr/jit/compiler.h

```diff
@@ -9424,7 +9424,7 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 #define DEFAULT_MAX_LOOPSIZE_FOR_ALIGN DEFAULT_ALIGN_LOOP_BOUNDARY * 3
 
 // By default only single iteration loops will be unrolled
```

Comment:
Does this comment need updating?

Reply:
Yes, oops. Totally missed the comment

Will get up a fix

Reply:
https://github.com/dotnet/runtime/pull/81300